jax/_src/numpy/ufuncs.py

```diff
@@ -2557,8 +2557,14 @@ def _float_divmod(x1: ArrayLike, x2: ArrayLike) -> tuple[Array, Array]:
 
   ind = lax.bitwise_and(mod != 0, lax.sign(x2) != lax.sign(mod))
   mod = lax.select(ind, mod + x2, mod)
+  mod = _where(lax.sign(x2) != lax.sign(mod), negative(mod), mod)
   div = lax.select(ind, div - _constant_like(div, 1), div)
 
+  div = _where(x2 == 0., lax.div(x1, x2), div)
+  negative_inputs = logical_and(lax.sign(x1)==-1, lax.sign(x2)==-1)
+  div = _where(logical_and(lax.gt(x1, x2), negative_inputs), negative(div), div)
+  div = _where(x1 == 0., div * lax.sign(x1), div)
```

Comment:
If we use `signbit` above then I think this may not be necessary

Reply:
Thanks! Done.